devenv/docker/blocks/mssql_tls/build/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mcr.microsoft.com/mssql/server:2019-CU8-ubuntu-18.04
```

Comment:
To limit code duplication, would it be possible to reuse and extend the mssql/build solution to support building an image supporting tls from the mssql_tls block?